frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -404,11 +337,18 @@ export default memo(function HeatMap({
     selectedCellTypes: filteredCellTypes,
   });
 
+  useHandleExpandedTissueIds({
+    filteredCellTypeIds,
+    filteredTissueIds,
+    displayedTissueIds,
+    dispatch,
+  });
+
   /**
    * All tissue cell types to render in YAxisCharts
    */
   const allTissueCellTypes = useMemo(() => {
-    return Object.values(tissuesByName)
+    return displayedTissues
```

Comment:
use `displayedTissues` instead of `Object.values(tissuesByName)` for a shorter array to iterate, since `displayedTissues` is filtered and doesn't contain tissues with 0 cell types